src/app/layout.tsx

```diff
@@ -0,0 +1,264 @@
+import '../styles/global.css'
+
+import { Toaster } from '@/components/ui/toaster'
+import { client } from '@/gql/client'
+import { siteDataQuery } from '@/gql/query'
+
+import { GeistSans } from 'geist/font/sans'
+import { Metadata } from 'next'
+import localFont from 'next/font/local'
+
+// Font files can be colocated inside of `pages`
+const menloFont = localFont({ src: '../lib/fonts/Menlo-Regular.woff', variable: '--font-menlo' })
+
+const DESCRIPTION =
+  'River is a permissionless protocol for social networking apps. Create dynamic on-chain spaces that feature memberships, reputation, and end-to-end message encryption.'
+const metaImageUrl = 'https://hnt-river-dev-portal.vercel.app/og-image.jpeg'
```

Comment:
Can we read this static asset from source instead of a url that may break or change ?

Reply:
Nice suggestion, done!